denoiser_project/Data/ingestion.py

```diff
@@ -17,6 +16,7 @@ def process_audio(file_path, output_dir):
             chunk_name = os.path.join(output_dir, f"{os.path.basename(file_path).split('.')[0]}_chunk{i}.wav")
```

Comment:
use this library "from pathlib import Path
file_path = Path(file_path)
chunk_name = output_dir / f"{file_path.stem}_chunk{i}.wav""